css-two-column-layout/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+ <div class="container-all">
+    <div class="row1"></div>
+      <div class="column-full">
+                <h1>SCALE, SPEED, AGILITY</h1>
+                <p>We've dedicated thousands of man hours to making healthcare institutions in the Helium</p>
+                <p>network run 10x for efficiently and deliver an unparallelled quality of care.</p></div>
```

Comment:
In general, especially for elements that are not text elements (such as the `div` here), you want closing tags to be on their own line, so it is easy to see where elements start and end. Your indentation here is spotty as well, which is probably not helping yo u to navigate your code easily. Formatting is super important!